src/main/java/hudson/plugins/ec2/SlaveTemplate.java

```diff
@@ -525,6 +547,14 @@ public int getMaxTotalUses() {
         return maxTotalUses;
     }
 
+    public List<NodePropertyDescriptor> getNodePropertyDescriptors() {
+        return NodePropertyDescriptor.for_(NodeProperty.all(), EC2AbstractSlave.class);
+    }
+
+    public DescribableList<NodeProperty<?>, NodePropertyDescriptor> getNodeProperties() {
+    	return Objects.requireNonNull(nodeProperties);
```

Comment:
This line seems to not compile could this be fixed up?

Reply:
It seems 9c26fdc32 broke it, I have rebased and fixed ^^